tool/jt.rb

```diff
@@ -716,6 +716,10 @@ def test_mri(*args)
     if args.delete('--openssl')
       include_pattern = "#{TRUFFLERUBY_DIR}/test/mri/tests/openssl/test_*.rb"
       exclude_file = "#{TRUFFLERUBY_DIR}/test/mri/openssl.exclude"
+    elsif args.delete('--cext')
```

Comment:
Maybe we should name this `--syslog`?
And if we have `--cexts` it would include openssl, etc as well?